Pipfile

```diff
@@ -23,6 +23,7 @@ simple-websocket = "==0.9"
 taipy-config = {git = "https://git@github.com/avaiga/taipy-config.git@develop"}
 tzlocal = "==3.0"
 gitignore-parser = "==0.1.1"
+twisted = "==22.10.0"
```

Comment:
Should this be optional ?

Reply:
I guess is could: only makes sense when running within Notebooks...
But then how do we address the versioning problem?

Reply:
We should keep this as required installation in pipfile since it directly affects the notebook side of things